nlmod/plot/plot.py

```diff
@@ -48,6 +52,40 @@ def modelgrid(ds, ax=None, **kwargs):
     return ax
 
 
+def modelextent(ds, dx=None, ax=None, rotated=True, **kwargs):
+    """Plot model extent.
+
+    Parameters
+    ----------
+    ds : xarray.Dataset
+        The dataset containing the data.
+    ax : matplotlib.axes.Axes, optional
+        The axes object to plot on. If not provided, a new figure and axes will be
+        created.
+    **kwargs
```

Comment:
Can you add `dx`  and `rotated` parameters to the docstring. Should we make `rotated` False by default?

Reply:
Added, I kept the rotated=True default for now. Deferring that decision to @rubencalje .

Reply:
I think @OnnoEbbens has a good point, as we have specified the following in the grid-rotation-notebook (https://nlmod.readthedocs.io/en/latest/examples/11_grid_rotation.html):
`the plot-methods in nlmod plot the grid in model-coordinates by default (can be overridden by the setting the parameter ‘rotated’ to True)`

Since then, new plot-methods have adapted a default of rotated=True. I will change these defaults to False